# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/reftest.list
# Commit: 8c70bc92ad36
# Full Hash: 8c70bc92ad36beb61f2cd6031e80ace3793fe53b
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r c6f1243c3ba4 -r 8c70bc92ad36 gfx/tests/reftest/reftest.list
--- a/gfx/tests/reftest/reftest.list	Wed Feb 23 20:42:19 2022 +0000
+++ b/gfx/tests/reftest/reftest.list	Wed Feb 23 20:49:26 2022 +0000
@@ -20,7 +20,7 @@
 == 1519754.html 1519754-ref.html
 skip-if(!asyncPan) == 1524261.html 1524261-ref.html
 fuzzy-if(!useDrawSnapshot,14-14,44-95) == 1524353.html 1524353-ref.html
-fuzzy-if(!useDrawSnapshot,2-7,17500-36908) == 1523776.html 1523776-ref.html
+== 1523776.html 1523776-ref.html
 == bug1523410-translate-scale-snap.html bug1523410-translate-scale-snap-ref.html
 == 1523080.html 1523080-ref.html
 == 1616444-same-color-different-paths.html 1616444-same-color-different-paths-ref.html